src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -203,12 +203,17 @@ public bool IsEnabledForMultiPlatformBuild(BuildScriptGeneratorContext scriptGen
             var packageJson = GetPackageJsonObject(sourceRepo, _logger);
             if (packageJson != null)
             {
-                var npmVersion = GetNpmVersion(packageJson);
+                string npmVersion = GetNpmVersion(packageJson);
+                _logger.LogDebug("GetNpmVersion returned {npmVersion}", npmVersion);
                 if (!string.IsNullOrEmpty(npmVersion))
                 {
                     toolsToVersion[NodeConstants.NpmToolName] = npmVersion;
                 }
             }
+            else
+            {
+                _logger.LogDebug("packageJson is null");
```

Comment:
adding more context would be help, e.g. "Not settings Node tools version since packageJson is null"